frontend/src/components/regions/UserActions/Presentation.jsx

```diff
@@ -3,28 +3,29 @@ import Paper from "@material-ui/core/Paper";
 import Grid from "@material-ui/core/Grid";
 import Loading from "../../widgets/Loading";
 import { makeStyles } from "@material-ui/core/styles";
-import DayLog from "./DayLog";
 import UserBurnDownChart from "./UserBurndownStats";
+import ActivityLog from "./ActivityLog";
+import Button from "../../widgets/Button";
 
 const useStyles = makeStyles((theme) => ({
   column: {
-    height: "85%", // TODO. Fit viewport
+    height: "100%", // TODO. Fit viewport
     overflowY: "scroll",
   },
 }));
 
 export default function Presentation({
   orderedDates,
-  actionLogByDate,
-  handleClickOpenProjectDetails,
   handleScroll,
   anyLoading,
   currentUserBurndownStats,
+  activityLogEntries,
+  fetchNextPages,
 }) {
   const classes = useStyles();
   return (
-    <div className={classes.column} onScroll={handleScroll}>
-      <Grid container>
+    <div className={classes.column}>
+      <Grid container onScroll={handleScroll}>
```

Comment:
move the onscroll handler back to where it was so that those who can paginate via scrolling can and those who can't can use the button